Recipely/Recipely/ProfileScreen/Cells/InfoCell/InfoTableViewCell.swift

```diff
@@ -0,0 +1,87 @@
+// InfoTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка с информацией о пользователе
+final class InfoTableViewCell: UITableViewCell {
+    // MARK: - Visual Components
+
+    private let avatarImageView = UIImageView()
+    private let fullNameLabel = UILabel()
+    private let changeButton = UIButton()
+
+    // MARK: - Public Properties
+
+    var onTapHandler: (() -> ())?
+
+    // MARK: - Initializers
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        configureAvatar()
+        configureLabel()
+        configureButton()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Public Methods
+
+    func configureCell(model: Info, tapButton: (() -> ())?) {
```

Comment:
```suggestion
    func configureCell(model: Info, tapButton: (() -> ())?) {
```
в (() -> ())? тайп Элиас